substratevm/src/com.oracle.svm.core.jdk.jfr.test/src/com/oracle/svm/core/jdk/jfr/test/event/TestSingleEventClean.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020, 2021, Red Hat, Inc.
+ *
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This file is part of the Red Hat GraalVM Testing Suite (the suite).
+ *
+ * The suite is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ * Foobar is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Foobar.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.oracle.svm.core.jdk.jfr.test.event;
+
+import com.oracle.svm.core.jdk.jfr.test.utils.events.StringEvent;
+
+import jdk.jfr.consumer.RecordedEvent;
+import jdk.jfr.consumer.RecordingFile;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class TestSingleEventClean {
+
+    @Test
+    public void test() throws Exception {
+        long s0 = System.currentTimeMillis();
+        StringEvent event = new StringEvent();
+        event.message = "Event has been generated!";
+        event.commit();
+
+        long d0 = System.currentTimeMillis() - s0;
+//        System.out.println("elapsed:" + d0);
```

Comment:
The elapsed stuff is removed from the other tests but is still here. Should it be removed as well? Everything else looks fine to me!

Reply:
To be honest, I don't understand what this test is about, and it doesn't actually test anything. Maybe remove the whole test?

Reply:
Right. The purpose of this test is to verify behavior of custom event code based on command-line arguments. A recording can be started via the Java API as in the other classes, but also via command-line such as `$ java -XX:StartFlightRecording=dumponexit=true`. So when running this manually, with no relevant command line flags, there should be no JFR output, while with relevant command line flags, there should be a JFR file. In the native image case, this would be `./blob` versus `./blob -XX:StartFlightRecording=dumponexit=true`

I think its okay to remove it from this junit test setup for now until we figure out how we want these tests to be run. I would imagine placing them under a different package and having scripts that run subsets with relevant flags or not. The results differ based on the flags so verification in the Java code via JUnit API will need further thought.

Reply:
Ok, interesting. Yes, I'll cut out this particular test then (tomorrow). Maybe in the future TestSingleEven.java (and all other tests) would replace it altogether.

Reply:
Sounds good, thank you.